plugin/plugin.go

```diff
@@ -54,8 +54,8 @@ type Type string
 func (t Type) String() string { return string(t) }
 
 const (
-	// AllPlugins declares that the plugin should be initialized after all others.
-	AllPlugins Type = "*"
```

Comment:
This was a hack to ensure that the introspection plugin loads last. There may be better way to enforce that but just make sure it appears last. I suppose hiding it with internal is sufficient.

Reply:
Are there any packages externally using this type?  I grepped and didn't find anything internally so I felt it was safe to change and give it a type instead of `*`.  It should function the same as before.